app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -24,7 +24,7 @@ def user_info
     info.merge!(x509_attributes) if scoper.x509_scopes_requested?
     info[:verified_at] = verified_at if scoper.verified_at_requested?
     if identity.vtr.nil?
-      info[:ial] = asserted_ial_value
+      info[:ial] = authn_context_resolver.asserted_ial_acr
```

Comment:
we still need a small amount of code in this class -- `authn_context_resolver` will bomb out if `identity` doesn't exist.

Reply:
Are you seeing a case that is not protected by this line in `AuthnContextResolver#asserted_ial_acr`:
```
return Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF unless user.active_profile.present?
```

Reply:
mm, when lauren and i paired on this, it failed when we tried a similar thing, but if the tests are all passing, i'm probably misremembering. sorry!